src/group.js

```diff
@@ -231,9 +231,20 @@ var _ = Mavo.Group = $.Class({
 		this.propagate(obj => {
 			var propertyData = data[obj.property];
 
-			if (obj.alias && data[obj.alias] !== undefined) {
-				copy = copy || $.extend({}, data);
-				propertyData = data[obj.alias];
+			// find first alias with data, load that data, and set to be copied
+			if (obj.alias) {
+				aliasesArr = obj.alias.split(" ");
+				var i = 0;
+				while (i<aliasesArr.length){
```

Comment:
Is there a reason this while is not a for loop? It looks like a textbook for loop converted to use while…

Also, please install ESLint (and an ESLint plugin for your editor) which will help you follow the code style of the project. :)